monitors/didyoumean.js

```diff
@@ -16,8 +16,8 @@ exports.run = async (client, msg) => {
     }));
     distances.sort((a, b) => (a.score < b.score ? 1 : -1));
     if (distances[0] && distances[0].dist <= 1) {
-      const message = await msg.channel.send(`|\`❔\`| Did you mean \`${conf.prefix + distances[0].cmd}\`?`);
-      setTimeout(() => { if (message.deletable) message.delete(); }, 10000);
+      const message = await msg.channel.send(`|\`❔\`| Did you mean \`${prefix + distances[0].cmd}\`?`);
+      if (message.deletable) message.delete({ timeout: 10000 });
```

Comment:
message.deletable becomes redundant there. You should await and catch that message.delete.